test/Microsoft.Dnx.Testing.CommonUtils/Directory/Dir.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```

Comment:
Just a reminder please sort and remove unused `using` statements. (Because these four `using` statements are always automatically added. So I'm just wondering.)